server/athenian/api/models/__init__.py

```diff
@@ -78,6 +80,42 @@ def create_base() -> BaseType:
     return declarative_base(cls=(Refreshable, Explodable))
 
 
+class SmartColumn(Column):
+    """Column that automatically converts `IN` and `NOT IN` to `ANY` and `!= ALL` repsecitvely"""
+
+    # 32767 is the maximum value for postgres:
+    #   - https://sentry.io/organizations/athenianco/issues/1785993425/
+    MAX_ALLOWED_QUERY_ARGUMENTS = 32767
+    # Anyway we cannot use exactly the upper limit because the length of the values used for
+    # `IN` and `NOT IN` could be near the the maximum allowed, but it often comes with other
+    # clauses that could make the total number of arguments bindings overflow the limit.
+    # This switch also affects performances:
+    #   - https://blog.jooq.org/2017/03/30/sql-in-predicate-with-in-list-or-with-array-which-is-faster/ # noqa: E501
+    IN_AUTO_ANY_THRESHOLD = 16383  # This specific value doesn't have any specific meaning
```

Comment:
I'd like to also run all the metrics test with this value overridden to `0`, wdyt?

Reply:
IMHO we don't need two thresholds as soon as we set the main one to 100.